docs/datasheets/augmented_dagw.md

```diff
@@ -0,0 +1,96 @@
+```
+- A sample from Scandi-Reddit:
+```
+{'text': 'Bergen er ødelagt. Det er ikke moro mer....', 'id': '0', 'source': 'scandi-reddit', 'created': '2005-12-01T00:00:00.000Z, 2022-11-01T00:00:00.000Z', 'added': '2024-04-17T12:50:03.000Z', 'metadata': {'language': 'da', 'language_confidence': 0.7472341657, 'subreddit': 'Norway'}}
+```
+- A sample from Nordjylland-News:
+```
+{'id': 'nordjylland-news0', 'text': 'Opdatering: Manden er nu fundet af Nordjyllands Po...', 'source': 'TV2 Nord', 'added': '2024-03-15T14:35:19.000Z', 'created': '2024-03-15T14:35:19.000Z, 2025-03-15T14:35:19.000Z', 'metadata': {'summary': 'Nye oplysninger i sagen om en forsvunden mand har endnu en gang fået politiet til at henvende sig til borgerne.', 'text_len': 1739, 'summary_len': 111}}
+```
+
+## Data Collection
+Here we list the python scripts for collecting, converting and compressing each sub-dataset.
+
+- DAGW: https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/main/data-processing/scripts/convert_dagw_to_jsonlgz.py
+- MeMo: https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/main/data-processing/scripts/memo/convert_memo_to_jsonlgz.py
+- Scandi-Reddit: https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/main/data-processing/scripts/convert_scandi_reddit_to_jsonlgz.py
+- Nordjylland-News: https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/main/data-processing/scripts/convert_nordjyllandnews_to_jsonlgz.py
+## Data Cleaning
+
+1. First, we filtered out all the empty documents (based on `"text"`: either *empty* or totally *white space*) in DAGW: 162051 out of 673211 empty docs found in total.
+2. After combining all 4 components, we applied deduplication and URL block list to the augmented DAGW following this [filtering pipeline](https://github.com/centre-for-humanities-computing/danish-foundation-models/blob/main/data-processing/configs/2024-v1/README.md) with [Dolma](https://github.com/allenai/dolma).
```

Comment:
It does not hurt to delete empty documents, but I don't think we should do line level deduplication and URL blocking at this stage.

Reply:
Ok, Peter, do you mean we should remove this second cleaning step at this stage?

Reply:
Yes, I don't think we should do the second cleaning step because it might mess up some of the content.